### PR TITLE
Fix /redir to global room message

### PR DESCRIPTION
**Pull request by Morfent**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -740,7 +740,7 @@ var commands = exports.commands = {
 			return this.sendReply("User " + this.targetUsername + " not found.");
 		}
 		if (Rooms.rooms[targetRoom.id].users[targetUser.userid]) {
-			return this.sendReply("User " + targetUser.name + " is already in the room " + targetRoom.title + "!");
+			return this.sendReply("User " + targetUser.name + " is already in the room " + (targetRoom.title || "global") + "!");
```
Comment (kotarou3) [inline commands.js]: `(targetRoom.title || targetRoom.id)` sounds like a better solution for this

Comment (Morfent) [inline commands.js]: I guess it would be safer than just assuming that the room would always be global.

**Comment by Zarel** [comment]
I still don't get it. What is this doing?

**Comment by Morfent** [comment]
"User is already in room undefined!"

**Comment by Zarel** [comment]
That message would never happen. It would be

"User Morfent is already in room undefined!"

Your commit just changes it to

"User Morfent is already in room morfent!"

Which is equally silly.

**Comment by Zarel** [comment]
Also, what kind of command would produce that message?

**Comment by Zarel** [comment]
Is it `/redir morfent, global`? If so, we probably want a custom message like "You can't redirect to the global room."

File: commands.js [hunk 2]
```diff
@@ -740,7 +740,7 @@ var commands = exports.commands = {
 			return this.sendReply("User " + this.targetUsername + " not found.");
 		}
 		if (Rooms.rooms[targetRoom.id].users[targetUser.userid]) {
-			return this.sendReply("User " + targetUser.name + " is already in the room " + targetRoom.title + "!");
+			return this.sendReply("User " + targetUser.name + " is already in the room " + (targetRoom.title || targetUser.id) + "!");
```
Comment (codelegend) [inline commands.js]: Did you mean `targetRoom.id` and not `targetUser.id`?

Comment (Morfent) [inline commands.js]: Yes, that was what I intended.

**Comment by Morfent** [comment]
Added a separate message if ``targetRom.id === "global"``

**Pull request merged**